MyCryptoHQ/MyCrypto
# Re-set address state when wallet changes.

Closes #1103.

### Description

Implements `componentWillReceiveProps` in the send message component to update address in state when the wallet changes, so that it reflects the latest wallet, not the first.

### Steps to Test

1. Unlock wallet A on Send, copy address
2. Change to wallet B via the button in the top right of the Send page
3. Paste in wallet A's address
4. Confirm you get no message about sending to yourself
5. Copy and paste in wallet B's address
6. Confirm you do get a message about sending to yourself
